src/support/slack/commands/onboard.js

```diff
@@ -47,10 +47,12 @@ function OnboardCommand(context) {
     name: 'Onboard Site(s)',
     description: 'Onboards a new site (or batch of sites from CSV) to Success Studio.',
     phrases: PHRASES,
-    usageText: `${PHRASES[0]} {site} {imsOrgId} [profile]`, // todo: add usageText for batch onboarding with file
+    usageText: `${PHRASES[0]} {site} [imsOrgId] [profile] [Optional: workflowWaitTime]`,
```

Comment:
I think the square brackets indicate that it's an optional param
```suggestion
    usageText: `${PHRASES[0]} {site} [imsOrgId] [profile] [workflowWaitTime]`,
```